style and grammar edits on tutorials (#4404)
getsentry/sentry-docs: src/docs/product/sentry-basics/guides/integrate-frontend/configure-scms.mdx

description: "Learn more about how Sentry uses commit metadata from your source code repositories to help you resolve your issues faster."
---

> Sentry uses commit metadata from your source code repositories to help you resolve your issues faster. This is done by suggesting _suspect commits_ that might have introduced an error right in your **Issue Details** page. It also allows Sentry to display _suggested assignees_ — the list of authors of those commits — and suggest their assignment to resolve the issue.

Author: ```suggestion
Sentry uses commit metadata from your source code repositories to help you resolve your issues faster. This is done by suggesting _suspect commits_ that might have introduced an error right in your **Issue Details** page. It also allows Sentry to display _suggested assignees_ — the list of the authors of those commits — and suggest their assignment to resolve the issue.
```

Now that you've created a release, you can tell Sentry which commits are associated with this latest version of your code; this is called _commit tracking_.

## Step 1: Integrate your GitHub account and repositories

1. To integrate GitHub with your Sentry org, follow the instructions in [our GitHub documentation](/product/integrations/source-code-mgmt/github/).

2. In the last step, add the `frontend-monitoring` repository from your GitHub account.

   ![Add project repository](configure-scms-01.png)

## Step 2: Set commit tracking

Now that you've set up releases in Sentry as part of your CI/CD process and integrated your source code repositories, you can associate commits from your linked repository to your releases.

<Note> 

> In the demo project, we use a Makefile to handle our build-related tasks. If you are not using the provided React demo code and do not have a Makefile, you could optionally run the `sentry-cli` commands used in this tutorial directly from the `command line` or integrate the commands into the relevant build script.

Author: ```suggestion
In the demo project, we use a `Makefile` to handle our build-related tasks. If you are not using the provided React demo code and do not have a `Makefile`, you can run the `sentry-cli` commands used in this tutorial directly from the `command line` or integrate the commands into the relevant build script.
```

       sentry-cli releases -o $(SENTRY_ORG) -p $(SENTRY_PROJECT) set-commits --auto $(REACT_APP_RELEASE_VERSION)
   ```

   > The command associates commits with the release. The _auto_ flag automatically determines the repository name, and associates commits between the previous release’s commit and the current head commit with the release.

3. The new target, `associate_commits`, will be invoked as part of the `setup_release` target. Add it at the end:

   ```bash
   setup_release: create_release upload_sourcemaps associate_commits
   ```

   Your `Makefile` should look like this:

   ![Updated Makefile](configure-scms-02.png)

4. If your terminal is still serving the demo app on localhost, press `^C` to shut it down.

5. Build, serve, and restart the project on your localhost by running:

   ```bash
   > npm run deploy
   ```

>    In the terminal log, notice that the `sentry-cli` identified the GitHub repository.

Author: ```suggestion
  > In the terminal log, notice that the `sentry-cli` identified the GitHub repository.
```
